integrations/pjrt/requirements.txt

```diff
@@ -1,4 +1,4 @@
 -f https://iree.dev/pip-release-links.html
 -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html
 iree-base-compiler==3.0.0
-jaxlib==0.4.17.dev20230922
+jaxlib==0.4.17
```

Comment:
SInce 0.4.17 is already released, I don't think we need to depend on a nightly version here.

Also, it will be converted to `jaxlib>=0.4.17` in setup.py, so I don't upgrade it to the latest version. Any suggestion on this? 🤔

Reply:
That's fine for now. If we start publishing this package to pypi (or github releases for nightly packages) then we can iterate on the setup.py to pin more deliberately.

Reply:
Got it! Maybe I'll first revert this change and we can change it in a seperate PR in the future : )